docs/support-bundle-guide.md

```diff
@@ -0,0 +1,183 @@
+```
+
+Then we could apply the following YAML files.
+The first one is to collect support bundle on Antrea Nodes "kube-worker" and "kube-master".
+The second one is to collect support bundle on ExternalNode "vm".
+
+```yaml
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-kube-nodes
+spec:
+  nodes:
+    nodeNames: // empty list means all the Nodes
+      - kube-master
+      - kube-worker
+  expirationMinutes: 10 // expirationMinutes is the requested duration of validity of the SupportBundleCollection. A SupportBundleCollection will be marked as Failed if it does not finish before expiration.
+  fileServer:
+    url: sftp://yourtestdomain.com:22/root/test 
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-vm-external-node
+spec:
+  externalNodes:
+    nodeNames: // empty list means all the ExternalNodes in namespace
+      - vm
+    namespace: vm-ns
+  fileServer:
+    url: yourtestdomain.com:22/root/test //scheme sftp can be omitted
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+You can check the status of SupportBundleCollection by running
+`kubectl get supportbundlecollections NameOfSupportBundleCollection -ojson`. The following example
+shows a successful creation of a SupportBundleCollection. `collectedNodes` and `desiredNodes`
+should match the number of Nodes/ExternalNodes you want to collect support bundle. There is also
+an entry of "Completed" status in the "conditions" list.
+
+```bash
+$ kubectl get supportbundlecollections collect-support-bundle-for-kube-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+To access the files, SSH into yourtestdomain.com with port 22. Three tarballs,
+two for `collect-support-bundle-for-kube-nodes` and one for
+`collect-support-bundle-for-vm-external-node` are expected to be shown in `/root/test` folder.
+Run the `tar xvf TARBALL_NAME` command and you will get the files,
+each with different collect items.
```

Comment:
```suggestion
The output contains three tarballs. To access these files, please SSH into the ftp server `yourtestdomain.com` with port 22. There will be two tarballs for `collect-support-bundle-for-kube-nodes` and one for
`collect-support-bundle-for-vm-external-node` showing in `/root/test` folder.
Run the `tar xvf TARBALL_NAME` command and you will get extracted files.
Each tarball contains different collect items:
```

I am not sure I got the idea about ` two tarballs for `collect-support-bundle-for-kube-nodes` `, I suppose you mean there will be one tarball for a K8s Node, and you have two Nodes, so there are two tarballs?
I feel it's better to show the output of `/root/test` to let user understand the output.

Reply:
@wenyingd do you have this kind of information?